modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -1129,15 +1162,20 @@ public abstract class Base${schemaName}ResourceImpl
 		(${type})parameters.get("${value}")
 	<#else>
 		<#if type?contains("java.lang.Boolean")>
-			Boolean.parseBoolean(
+			<#assign parserMethodDataTypes = parserMethodDataTypes + ["Boolean"] />
+				_parseBoolean(
 		<#elseif type?contains("java.util.Date")>
-			new java.util.Date(
+			<#assign parserMethodDataTypes = parserMethodDataTypes + ["Date"] />
+				_getParsedDate(
 		<#elseif type?contains("java.lang.Double")>
-			Double.parseDouble(
+			<#assign parserMethodDataTypes = parserMethodDataTypes + ["Double"] />
+				_parseDouble(
```

Comment:
why is there `parseDouble` and `_getParsedDate`? Shouldn't they follow the same naming pattern?

Reply:
I agree, we must keep the same pattern

Reply:
And I don't see any `_getParsedDate` method

Reply:
I've changed the name becouse there are some classes that already have a private _parseDate that adapts the format of a date. Maybe we can change all the parser format?